docs/operator-guide/troubleshooting/troubleshoot-stages.md

```diff
@@ -0,0 +1,38 @@
+# Application Is Not Deployed
+
+## Problem
+
+In the stage details page, the application status is unknown. The `deploy` button desn't work when attempting to create a stage.
+
+  !![Stage is not synced](../../assets/operator-guide/stage_is_not_synced.png "Stage is not synced")
+
+## Cause
+
+The `AppProject` custom resource may not created.
+
+## Solution
+
+1. Check the AppProject Resource:
+
+  ```bash
+     kubectl get crd appprojects.argoproj.io -n argocd
+  ```
+
+  It is supposed there are no resources or they are deployed incorectly.
+
+2. Create an `AppProject` custom resource by following the steps below:
+
+3. Create the `AppProject` custom resource. Please refer to the [Argo CD Integration](../../operator-guide/argocd-integration.md#argo-cd-configuration) page, (step 4).
+
+4. Apply the resource using the `kubectl apply` command and wait until the application is synced with Argo CD
+
+  !!! note
+      By default, it takes 5-10 minutes to take affect but it may vary depending on your personal platform configuration.
+
+5. Check the status of the application:
+
+  !![Stage is not synced](../../assets/operator-guide/stage_is_not_synced.png "Stage is not synced")
```

Comment:
Can we replace the image with the correct image?

Reply:
Done